### DISCUSSION TITLE
DIsable swiping to the next or prev slide when swiper is zoomed

### DISCUSSION THREAD

**Discussion by linuxonrails** [nolimits4web/swiper#5854] [Ideas]
> DIsable swiping next prev slide when swiper is zoomed #4280 https://github.com/nolimits4web/swiper/issues/4280
>
> If swiper slide is zoomed, i need to disable prev next slide swiping, cuz thats unreal to use zoom. U try to zoom, but swiper go to next slide. How to disable swiping next prev when swiper is zooming?

**Comment by dinhkham** [comment]
Looking forward to the answer with ReactJs :D
